app/src/main/java/com/example/parkingsystem/fragment/ListenerDialogFragment.java

```diff
@@ -0,0 +1,7 @@
+package com.example.parkingsystem.fragment;
+
+import java.io.Serializable;
+
+public interface ListenerDialogFragment extends Serializable {
```

Comment:
why do you need the Serializable here?

Reply:
I had to do the serializable, because otherwise I could not put it in the Bandle my interface